_includes/docs/devices-library/index.md

```diff
@@ -97,15 +97,15 @@
     }
 
     function reportSearchInput(searchText) {
-
-        if (!ga.hasOwnProperty("loaded") || ga.loaded !== true) {
-            return;
+        if (typeof window.dataLayer === "undefined" || !window.dataLayer.push) {
```

Comment:
I strongly suggest moving this expression into a separate JS function because it is used 8 times in the PR.